backend/packages/Upgrade/src/loaders/typeormLoader.ts

```diff
@@ -6,14 +6,55 @@ import { SERVER_ERROR } from 'upgrade_types';
 
 export const typeormLoader: MicroframeworkLoader = async (settings: MicroframeworkSettings | undefined) => {
   const loadedConnectionOptions = await getConnectionOptions();
-
+  const loadedSlaveConnectionOptions = await getConnectionOptions();
   const connectionOptions: ConnectionOptions = Object.assign(loadedConnectionOptions, {
+    name: 'default',
+    type: env.db.type, // See createConnection options for valid types
+    host: '',
+    port: '',
+    username: '',
+    password: '',
+    database: '',
+    replication: {
+      master: {
+        host: env.db.host,
+        port: env.db.port,
+        username: env.db.username,
+        password: env.db.password,
+        database: env.db.database,
+      },
+      slaves: [],
+    },
+    synchronize: env.db.synchronize,
+    logging: env.db.logging,
+    maxQueryExecutionTime: env.db.maxQueryExecutionTime,
+    entities: env.app.dirs.entities,
+    migrations: env.app.dirs.migrations,
+  });
+  const slaveConnectionOptions: ConnectionOptions = Object.assign(loadedSlaveConnectionOptions, {
+    name: 'export',
```

Comment:
Can you create an enum for the connection name because having hardcoded string will create trouble in the future

Reply:
done